include/range/v3/view/adaptor.hpp

```diff
@@ -251,7 +251,16 @@ namespace ranges
         struct basic_adaptor_mixin : basic_mixin<adaptor_cursor>
         {
             basic_adaptor_mixin() = default;
+            #ifndef _MSC_VER
             using basic_mixin<adaptor_cursor>::basic_mixin;
+            #else
+            explicit constexpr basic_adaptor_mixin(adaptor_cursor && cur)
+              : basic_mixin<adaptor_cursor>(static_cast<adaptor_cursor &&>(cur))
+            {}
+            explicit constexpr basic_adaptor_mixin(adaptor_cursor const & cur)
+              : basic_mixin<adaptor_cursor>(cur)
+            {}
+            #endif
```

Comment:
@CaseyCarter This is an example of where I needed to avoid inheriting constructors or else msvc would ICE in c++17-mode.